workspaces/jupyterlab-r/install.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -ex
+
+# Install all R dependencies.
+xargs mamba install --yes < /requirements.txt
```

Comment:
This strikes me as strange, this looks like it's masquerading as a Python `requirements.txt` file even though it isn't actually. Is there a compelling reason to do it like this? Otherwise you're probably better off just inlining the dependencies here:

```suggestion
xargs mamba install --yes r-base==4.4.3 r-irkernel==1.3.2
```

Reply:
That was from before I knew that you couldn't increment non-python packages